app/src/main/java/org/systers/mentorship/viewmodels/TaskViewModel.kt

```diff
@@ -26,7 +68,14 @@ class TasksViewModel: ViewModel() {
      * @param taskName title of the new task
      */
     fun addTask(taskName: String) {
-        taskList.add(taskName)
+        taskList.add(
```

Comment:
this is yet another task for testing right?

Reply:
Yup, the function is just a dummy